Resolved Bug 1370 : Design System > Components > Login > Default > Fixed Language Dropdown

Dipak7028 opened the pull request:
## Description

>Resolve Components > Login > Language Dropdown positioning issue
>Resolve Elements > Dropdown List > Dropdown List positioning issue
>Resolved Element level problem of Dropdownlist in dropdown.scss 
>Able to resolve placement issue of Select Language

-**Resolved positioning issue in dropdown list element**
-**Changed the dropdown.scss file**


## Screenshots
![image](https://github.com/user-attachments/assets/260bc28c-ed1a-4513-a9d9-efda85536047)
![image](https://github.com/user-attachments/assets/a55945fe-fa81-4272-b852-5fff1c50e9f4)
![image](https://github.com/user-attachments/assets/3fc62575-d338-48ff-8a76-d71cce2d8523)


## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

alok-nakhate reviewed (approved):
Changes are reviewed and approved.

Dipak7028 committed:
Resolved Bug 1370 : Design System > Components > Login > Default > Fixed Language Dropdown

Resolved Element level problem of Dropdownlist in dropdown.scss
Able to resolve placement issue of Select Language

supriyartambewaiin reviewed (approved):
Reviewed and Approved
